src/cnab.rs

```diff
@@ -265,3 +278,19 @@ pub struct Destination {
     /// The fully qualified path to the destination file
     pub path: Option<PathBuf>,
 }
+
+/// A value that is produced by running an invocation image
+///
+/// Complies to CNAB Core 1.0
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Output {
+    /// An optional exhaustive list of actions producing this output
+    pub apply_to: Option<String>,
```

Comment:
This should be `Option<Vec<String>>` - see https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md#outputs

Reply:
You are correct. Thanks